libs/components/toast/testing/src/modules/toast/toaster-harness.spec.ts

```diff
@@ -19,8 +19,12 @@ import { SkyToasterHarness } from './toaster-harness';
 class TestComponent {
   public toastSvc = inject(SkyToastService);
 
-  public openToast(message: string, type: SkyToastType): void {
-    this.toastSvc.openMessage(message, { type: type });
+  public openToast(
+    message: string,
+    type: SkyToastType,
+    autoclose?: boolean,
+  ): void {
+    this.toastSvc.openMessage(message, { type: type, autoClose: autoclose });
```

Comment:
Rather than adding a `boolean` parameter, could the second parameter be either `SkyToastType` (since that's what we have currently) or an optional `SkyToastConfig` to match the `SkyToastService` API?

Alternatively, it'd be reasonable for this method to not accommodate `autoclose`.